### ISSUE TITLE
Server stop

### ISSUE THREAD

**Issue by HeXXitTheGamer** [PocketMine/PocketMine-MP#4273]
### Issue description

Write a short description about the issue
### Steps to reproduce the issue

1. ...
2. ...
### OS and versions

- PocketMine-MP:
- PHP:
- OS:
### Crashdump, backtrace or other files

[PocketMine] Server is starting...
[PocketMine] Server was started.
[Server] <!DOCTYPE html>
[Server] 
[Server] 
[Server] 
[Server]  <html class="no-js" lang="en-US"> 
[Server] <head>
[Server] <meta http-equiv="refresh" content="0">
[Server] <meta http-equiv="set-cookie" content="cf_use_ob=80; expires=Fri, 26-Aug-16 05:56:07 GMT; path=/">
[Server] <meta http-equiv="set-cookie" content="cf_ob_info=521:2d85092dd3982938:OTP; expires=Fri, 26-Aug-16 05:56:07 GMT; path=/">
[Server] <title>jenkins.pocketmine.net | 521: Web server is down</title>
[Server] <meta charset="UTF-8"/>
[Server] <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
[Server] <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1"/>
[Server] <meta name="robots" content="noindex, nofollow"/>
[Server] <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1"/>
[Server] <script type="text/javascript">
[Server] //<![CDATA[
[Server] try{if (!window.CloudFlare) {var CloudFlare=[{verbose:0,p:0,byc:0,owlid:"cf",bag2:1,mirage2:0,oracle:0,paths:{cloudflare:"/cdn-cgi/nexp/dok3v=1613a3a185/"},atok:"845d4749af25265dd7714cc06ac7da2b",petok:"93c6095d0804f134cee165775a97af949dbe72a9-1472190937-1800",adblock:1,betok:"6f1a605879a41f3588ffcaabcfff1ddbe9ba9ab1-1472190937-120",zone:"pocketmine.net",rocket:"0",apps:{},sha2test:0}];!function(a,b){a=document.createElement("script"),b=document.getElementsByTagName("script")[0],a.async=!0,a.src="//ajax.cloudflare.com/cdn-cgi/nexp/dok3v=0489c402f5/cloudflare.min.js",b.parentNode.insertBefore(a,b)}()}}catch(e){};
[Server] //]]>
[Server] </script>
[Server] <link rel="stylesheet" id="cf_styles-css" href="/cdn-cgi/styles/cf.errors.css" type="text/css" media="screen,projection"/>
[Server] 
[Server] <style type="text/css">body{margin:0;padding:0}</style>
[Server] 
[Server] <script type="text/javascript" src="/cdn-cgi/scripts/zepto.min.js"></script>
[Server] <script type="text/javascript" src="/cdn-cgi/scripts/cf.common.js"></script>
[Server] </head>
[Server] <body>
[Server] <div id="cf-wrapper">
[Server] <div id="cf-error-details" class="cf-error-details-wrapper">
[Server] <div class="cf-wrapper cf-error-overview">
[Server] <h1>
[Server] <span class="cf-error-type" data-translate="error">Error</span>
[Server] <span class="cf-error-code">521</span>
[Server] <small class="heading-ray-id">Ray ID: 2d85092dd3982938 &bull; 2016-08-26 05:55:37 UTC</small>
[Server] </h1>
[Server] <h2 class="cf-subheadline" data-translate="error_desc">Web server is down</h2>
[Server] </div> 
[Server] <div class="cf-section cf-highlight cf-status-display">
[Server] <div class="cf-wrapper">
[Server] <div class="cf-columns cols-3">
[Server] <div id="cf-browser-status" class="cf-column cf-status-item cf-browser-status ">
[Server] <div class="cf-icon-error-container">
[Server] <i class="cf-icon cf-icon-browser"></i>
[Server] <i class="cf-icon-status cf-icon-ok"></i>
[Server] </div>
[Server] <span class="cf-status-desc" data-translate="browser_desc">You</span>
[Server] <h3 class="cf-status-name" data-translate="browser_label">Browser</h3>
[Server] <span class="cf-status-label" data-translate="browser_status_label">Working</span>
[Server] </div>
[Server] <div id="cf-cloudflare-status" class="cf-column cf-status-item cf-cloudflare-status ">
[Server] <div class="cf-icon-error-container">
[Server] <i class="cf-icon cf-icon-cloud"></i>
[Server] <i class="cf-icon-status cf-icon-ok"></i>
[Server] </div>
[Server] <span class="cf-status-desc" data-translate="cloud_desc">Bucharest</span>
[Server] <h3 class="cf-status-name" data-translate="cloud_label">CloudFlare</h3>
[Server] <span class="cf-status-label" data-translate="cloud_status_label">Working</span>
[Server] </div>
[Server] <div id="cf-host-status" class="cf-column cf-status-item cf-host-status cf-error-source">
[Server] <div class="cf-icon-error-container">
[Server] <i class="cf-icon cf-icon-server"></i>
[Server] <i class="cf-icon-status cf-icon-error"></i>
[Server] </div>
[Server] <span class="cf-status-desc" data-translate="server_desc">jenkins.pocketmine.net</span>
[Server] <h3 class="cf-status-name" data-translate="server_label">Host</h3>
[Server] <span class="cf-status-label" data-translate="server_status_label">Error</span>
[Server] </div>
[Server] </div>
[Server] </div>
[Server] </div> 
[Server] <div class="cf-section cf-wrapper">
[Server] <div class="cf-columns two">
[Server] <div class="cf-column">
[Server] <h2 data-translate="what_happened">What happened?</h2>
[Server] <p data-translate="not_returning_connection">The web server is not returning a connection. As a result, the web page is not displaying.</p>
[Server] </div>
[Server] <div class="cf-column">
[Server] <h2 data-translate="what_can_i_do">What can I do?</h2>
[Server] <h5 data-translate="if_website_visitor">If you are a visitor of this website:</h5>
[Server] <p data-translate="try_again_in_a_few">Please try again in a few minutes.</p>
[Server] <h5 data-translate="if_website_owner">If you are the owner of this website:</h5>
[Server] <p><span data-translate="521_owner_desc">Contact your hosting provider letting them know your web server is not responding.</span> <a data-translate="addtl_troubleshooting" href="https://support.cloudflare.com/hc/en-us/articles/200171916-Error-521">Additional troubleshooting information</a>.</p>
[Server] </div>
[Server] </div>
[Server] </div> 
[Server] <div class="cf-error-footer cf-wrapper">
[Server] <p>
[Server] <span class="cf-footer-item">CloudFlare Ray ID: <strong>2d85092dd3982938</strong></span>
[Server] <span class="cf-footer-separator">&bull;</span>
[Server] <span class="cf-footer-item"><span data-translate="your_ip">Your IP</span>: 92.83.120.100</span>
[Server] <span class="cf-footer-separator">&bull;</span>
[Server] <span class="cf-footer-item"><span data-translate="performance_security_by">Performance &amp; security by</span> <a data-orig-proto="https" data-orig-ref="www.cloudflare.com/5xx-error-landing?utm_source=error_footer" id="brand_link" target="_blank">CloudFlare</a></span>
[Server] </p>
[Server] </div> 
[Server] </div> 
[Server] </div> 
[PocketMine] Server was stopped.

**Comment by KairusDarkSeeker** [comment]
The Jenkins is down

**Comment by HeXXitTheGamer** [comment]
How to repair?? Plz

**Comment by GlitchPlayer** [comment]
Its pocketmine team problem m8. How did this even happened? tried to update via the console? GG m8 -.-

**Comment by HeXXitTheGamer** [comment]
I have added plugins to my server and this error appear in console

Pe 26.08.2016 13:51, "Glitch_Player" <notifications@github.com> a scris:

**Comment by TristonStuart** [comment]
Please provide more information on this, (os, version, php, steps you used to install, plugins you used, ect.)
----------------
This should be taken down for violation
If you are using mac i might be able to help :)
